local-test-operator.yml

```diff
@@ -150,13 +151,24 @@
             openshift_images_before: " {{ os_registry_is_result_before.stdout_lines }} + {{ os_pod_is_result_before.stdout_lines }}"
           when: not run_upstream|bool
 
+        - name: "Set the deployment start time"
+          shell: "date -u '+%Y-%m-%dT%H:%M:%SZ'"
+          register: start_time
+          when:
+            - run_deploy|bool
+            - not run_upstream|bool
+
         - name: "Deploy the operator on the OpenShift 4.0 with OLM"
           include_role:
             name: deploy_olm_operator
           when:
             - run_deploy|bool
             - not run_upstream|bool
 
+        - name: "Collect the names of the images modified during the deployment"
+          import_role:
+            name: collect_modified_images
+
```

Comment:
Should this be `include_role` instead of `import_role`?

Should this task also have the same `when` conditional as "Set the deployment start time" above since it depends on `start_time`? 

For clarity I suggest passing `start_time` explicitly by adding
```
vars:
  deployment_start_time: "{{ start_time }}"
```

Reply:
I had to import the role instead of including it so that the variables defined within the playbook would be available to the role - I guess passing manually is a bit cleaner